crates/client/sync/src/l2.rs

```diff
@@ -202,7 +203,7 @@ pub async fn sync(
     // starves the tokio worker
     let validation = BlockValidationContext {
         trust_transaction_hashes: false,
-        trust_global_tries: config.verify,
+        trust_global_tries: !config.verify,
```

Comment:
this is correct but this error must be corrected
```rust
 FetchConfig {
            gateway,
            feeder_gateway,
            chain_id,
            sound,
            verify: self.disable_root,
            api_key: self.gateway_key.clone(),
            sync_polling_interval: polling,
            n_blocks_to_sync: self.n_blocks_to_sync,
        }
```

verify: **!** self.disable_root,